torch/testing/_internal/common_methods_invocations.py

```diff
@@ -2673,7 +2673,7 @@ def sample_inputs_std_var(op_info, device, dtype, requires_grad, **kwargs):
         SampleInput(tensor_1d, kwargs=dict(dim=0, unbiased=False, keepdim=False)),
 
         SampleInput(tensor_nd, kwargs=dict(dim=(1,), correction=S // 2)),
-        SampleInput(tensor_nd, kwargs=dict(dim=None, correction=0, keepdim=True)),
+        SampleInput(tensor_nd, kwargs=dict(correction=0, keepdim=True)),
```

Comment:
Let's change the above "unbiased" uses to use "correction" and add a SampleInput with a correction factor that is not 0 or 1

Reply:
>Let's change the above "unbiased" uses to use "correction" 

The unbiased overloads are needed for backwards-compatibility so IMO they should be tested.

>add a SampleInput with a correction factor that is not 0 or 1

The line above this has `correction=S//2` which is 2.

Reply:
Cool